### ISSUE TITLE
notify users

### ISSUE THREAD

**Issue by abdi-osman** [SimkinsD/CMPT395-Term-Project#34]

**Comment by SimkinsD** [comment]
This isn't phrased as a user story; it is also very ambiguous. Notify users of what? Which users? Why?

**Comment by abdi-osman** [comment]
Which story?

**Comment by abdi-osman** [comment]
One of the stories is for changing your password and the other one is to
reset it if you forgot you password and can't login to your account

**Comment by SimkinsD** [comment]
I don't believe this was discussed as one of the objectives for our next sprint. If you check the sprint planning document, you may notice that this is not part of our goal; it was also not originally in the To Do list. Please discuss making additions to the To Do list with the group before adding them as it reflects poorly on our sprint planning if we start adding stories mid-sprint without good reason. Also, this is still not phrased as a user story. It is labelled "Facilitator" and should therefore be written from the perspective of a facilitator. I recommend adding additional labels to this issue, as in the title it references multiple types of users ("parents or staff"); the labels should include all users to which the story is directly relevant

**Comment by abdi-osman** [comment]
What's the facilitator again? Also I'm writing new user stories because the
only user story I saw for me was for changing user info I just though I
should do more.

**Comment by abdi-osman** [comment]
And also loging in as an individual in the family
